Content.Server/Botany/Systems/MutationSystem.cs

```diff
@@ -184,14 +136,8 @@ private void MutateFloat(ref float val, float min, float max, int bits, int tota
         val = valMutated;
     }
 
-    private void MutateInt(ref int val, int min, int max, int bits, int totalbits, float mult)
+    public void MutateInt(ref int val, int min, int max, int bits)
```

Comment:
Is the only caller in `PlantChangeStat`? If so, it might make sense to move this function there and keep it private.